Sources/StripeKit/Core Resources/Events/Event.swift

```diff
@@ -232,7 +232,112 @@ public enum EventObject: Codable {
         }
     }
     
-    public func encode(to encoder: Encoder) throws { }
+    public func encode(to encoder: Encoder) throws {
+        switch self {
+        case .account(let connectAccount):
```

Comment:
Also this is just happenstance but #254 added a new object that an `Event` should decode. Can you update the encode decode method to handle that? I think the `decode` was already added.